src/app/admin/components/case/case-file/case-additional-details/case-additional-details.component.spec.ts

```diff
@@ -0,0 +1,21 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { CaseAdditionalDetailsComponent } from './case-additional-details.component';
+
+describe('CaseAdditionalDetailsComponent', () => {
+  let component: CaseAdditionalDetailsComponent;
+  let fixture: ComponentFixture<CaseAdditionalDetailsComponent>;
+
+  beforeEach(async () => {
+    await TestBed.configureTestingModule({
+      imports: [CaseAdditionalDetailsComponent],
+    }).compileComponents();
+
+    fixture = TestBed.createComponent(CaseAdditionalDetailsComponent);
+    component = fixture.componentInstance;
+    fixture.detectChanges();
+  });
+
+  it('should create', () => {
+    expect(component).toBeTruthy();
+  });
```

Comment:
Maybe add some units tests around the displaying of certain fields that have logic in the template, such as "Case closed?"